### PR TITLE
Various Lunchboxes

### PR DESCRIPTION
**Pull request by DangerRevolution**
## About the PR

Adds various lunchboxes & their randomly-filled variants.

## Why / Balance

RP - Chef can make meals on-the-go for the Crew
Antags - An inconspicious carrying option

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image

**Changelog**
:cl:
- add: Added lunchboxes of various varieties, able to be found in the chef's cutlery vend.

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Food/Containers/lunchbox.yml [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+    - state: box-security
+    - state: box-open-security
+      map: ["openLayer"]
+  - type: Item
+    size: Normal
+    shape: null
+    sprite: DeltaV/Objects/Storage/Lunchbox/lunchbox_security.rsi
+    #heldPrefix: box-security
+    
+- type: entity
```
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Food/Containers/lunchbox.yml:157]: This is the only entity that loads.

**Comment by DangerRevolution** [comment]
Need a little assistance; none of the sprites for any of these except for Lunchbox Security Random load, and none of the in-hand sprites for any of them load (I know I've commented out the heldPrefix: but it doesn't work with it either)

**Comment by DEATHB4DEFEAT** [comment]
> [Port] [Kinda]

Is this a port or not? Where is it coming from?

**Comment by DangerRevolution** [comment]
> > [Port] [Kinda]
> 
> Is this a port or not? Where is it coming from?

Delta: https://github.com/DeltaV-Station/Delta-v/pull/716

It was never merged as Debug wanted all the spritesnib one folder and for the life of me I couldn't figure out how to do it, as doing it would break every sprite.

**Comment by DangerRevolution** [comment]
Debug managed to wizard them into one .rsi so just gonna copy it over when it's done :O

**Comment by DangerRevolution** [comment]
This should honestly just be merged from Delta's PR's by cherry-picking instead of a copypaste port. Anyways, I want to add the locker changes before this is PR-ed so gonna set this to draft

**Pull request closed without merging**